pkgs/by-name/ed/edopro/package.nix

```diff
@@ -0,0 +1,262 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, premake5
+, writeShellScriptBin
+, runCommandLocal
+, symlinkJoin
+, imagemagick
+, bzip2
+, curl
+, flac
+, fmt_9
+, freetype
+, libevent
+, libgit2
+, libGL
+, libGLU
+, libjpeg
+, libpng
+, libvorbis
+, libX11
+, libxkbcommon
+, libXxf86vm
+, lua5_3
+, nlohmann_json
+, noto-fonts-cjk-sans
+, openal
+, SDL2
+, sqlite
+, wayland
+, egl-wayland
+,
+}:
+let
+  edopro-version = "40.1.4";
+
+  # the assets repo has a font instead
+  font = "${noto-fonts-cjk-sans}/share/fonts/opentype/noto-cjk/NotoSansCJK-VF.otf.ttc";
+
+  covers_url = "";
+  fields_url = "";
+  pics_url = "http://images.ygoprodeck.com/images/cards/{}.jpg";
+
+  assets = {
+    pname = "assets";
+    version = edopro-version;
+
+    src = fetchTarball {
+      url = "https://github.com/ProjectIgnis/edopro-assets/releases/download/${edopro-version}/ProjectIgnis-EDOPro-${edopro-version}-linux.tar.gz";
+      sha256 = "sha256-vZhkWJ1ZoNEwdc5kM1S0hyXnWmupiTOanCi9DCuqw/k=";
+    };
```

Comment:
Please don't use the builtin `fetchTarball`, use Nixpkgs' `fetchurl` instead. (This is why the OfBorg eval is failing.)